NoMansLand/Assets/Scripts/SpaceshipManager.cs

```diff
@@ -19,17 +19,17 @@ public static class SpaceshipManager
 		static bool gameWon = false;
 		
 		// Time bonus earned when the player deposits a material
-    	static float TIME_BONUS = 15;
+    	static float TIME_BONUS = 20;
 
 		// The rust material pointer
 		static Material rust_material;		
 
 		// A mapping from spaceship part name to spacship part object
 		static Dictionary<string, SpaceshipPart> spaceshipParts =  
             new Dictionary<string, SpaceshipPart>(){
-				{"engine_frt_geo", new SpaceshipPart(1)},
-                {"engine_lft_geo", new SpaceshipPart(2)},
-                {"engine_rt_geo", new SpaceshipPart(3)},
+				{"engine_frt_geo", new SpaceshipPart(0)},
+                {"engine_lft_geo", new SpaceshipPart(0)},
+                {"engine_rt_geo", new SpaceshipPart(0)},
```

Comment:
Can you explain the reasoning behind this? Looks like a sequence of 1,2,3,4,5,6 became 0,0,0,4,5,6

Reply:
This is because the ship should have some damage initially - if the player quickly finds an item and returns it to the ship when only like 1 part dropped, then the game would end super fast. This at least makes them find a few pieces before possibly allowing them to win